cedar-policy-cli/src/lib.rs

```diff
@@ -495,33 +495,21 @@ pub fn validate(args: &ValidateArgs) -> CedarExitCode {
     let validator = Validator::new(schema);
     let result = validator.validate(&pset, mode);
 
-    let exit_code = if !result.validation_passed()
+    if !result.validation_passed()
         || (args.deny_warnings && !result.validation_passed_without_warnings())
     {
-        println!("Validation Failed");
+        println!(
+            "{:?}",
+            Report::new(result).wrap_err("policy set validation failed")
+        );
         CedarExitCode::ValidationFailure
     } else {
-        println!("Validation Passed");
+        println!(
+            "{:?}",
+            Report::new(result).wrap_err("policy set validation passed")
```

Comment:
why is this using `Report`? will this show as an error with miette's ❌?

Reply:
I set the severity to advice, so we get
![image](https://github.com/cedar-policy/cedar/assets/130772734/004b1428-05e1-44f4-8a05-277e3e26d9e3)

or, if there's a warning:

![image](https://github.com/cedar-policy/cedar/assets/130772734/8600d42b-9649-43bd-80b9-97118dffa8cd)

Reply:
Beautiful